project/src/pages/FilmDetailsPage/index.tsx

```diff
@@ -75,10 +71,12 @@ export const FilmDetailsPage = () => {
               </p>
 
               <div className="film-card__buttons">
-                <PlayFilmButton id={Number(id)}/>
-                <AddFilmFavoriteButton id={id}/>
                 {(id !== undefined) && (
```

Comment:
Let's remove this `(id !== undefined) &&`

And use this instead `const { id } = useParams() as { id: string };`